src/elements/content-sidebar/utils/isFileLargerThan.ts

```diff
@@ -0,0 +1,5 @@
+import { type BoxItem } from '../../../common/types/core';
+
+export function isFileLargerThan(file: BoxItem | null, breakpoint: number): boolean {
```

Comment:
[nit] the `breakpoint` is a bit vogue name. Something more verbose, like `breakpointSideInBytes` would document the behaviour much better.